packages/framework/src/Migrations/Version20190617070314.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20190617070314 extends AbstractMigration
+{
+    use MultidomainMigrationTrait;
+
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        foreach ($this->getAllDomainIds() as $domainId) {
+            $this->sql('INSERT INTO setting_values (name, domain_id, value, type) VALUES
```

Comment:
So there is a new setting for each domain, and the value is an empty string `''`. So I update, and run migrations. And now when I call `ElasticStructureManger::getCurrentProductIndexName()`, It will return this empty string, am I right? That doesn't seem like a correct behavior